pkg/sqlstore/pgstore/provider.go

```diff
@@ -0,0 +1,54 @@
+package pgstore
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
```

Comment:
```suggestion
	_ "github.com/lib/pq"
```
`lib/pq`'s last commit was in 2023 and has gone into maintenance. Let's use either `pgdriver` from bun or https://github.com/jackc/pgx.